README.md

```diff
@@ -8,6 +8,39 @@ effectively together with [nginx's autoindex module](http://nginx.org/en/docs/ht
 
 A sample nginx configuration is also included which mounts **file browser** under root (`/`) and mounts files to be listed under `/files` path. Hence is the `filesBaseUrl` under
 
+## Installation / Configuration
```

Comment:
I understand that in your use case a direct installation was probably a better choice, however in all these years I have been using this as an docker image (and that should remain the main focus). So it would be a good point to highlight that there are multiple options on how to use this and one is using the docker image and other is this direct installation.
Lets add a title called `## Usage` under which we will have the `### Using with Docker` and then this part under `### Direct Installation` after that.